### ISSUE TITLE
would you use react-native-big-list instead of flatlist?

### ISSUE THREAD

**Issue by z5614036** [flyerhq/react-native-chat-ui#29]
https://github.com/marcocesarato/react-native-big-list,   has a better perfermance.

**Comment by demchenkoalex** [comment]
Thanks, we will consider adding that!

**Comment by demchenkoalex** [comment]
As I understood `itemHeight` is required for this package, assuming all performance improvements are basically from the fact that items are of the same height. We have very dynamic items (messages, absolutely random height), and calculating text heights to supply to this package IMO will negate all performance improvements. Feel free to correct me if I missed something.

**Comment by Hirbod** [comment]
react-native-big-list is indeed not the right package for this. But [RecyclerList](https://github.com/Flipkart/recyclerlistview) is a bit painful to setup but is way faster than FlatList. But I would keep my eyes open for the upcoming "react-native-vision-list", which might be a game changer soon :)

**Comment by demchenkoalex** [comment]
I always though game changer will be Fabric and Turbo Modules :D 🤞 this year. And no dependencies needed :)

**Comment by Hirbod** [comment]
React Native Vision List will be some JSI magic ;). I am also waiting patiently for fabric and turbo modules to get rid of these painful lists in RN

**Comment by hirbod** [comment]
@demchenkoalex Shopify just released `FlashList` -  a super fast RLV implementation. Upgrading is actually super straight forward. Is this library still maintained and would you make it possible to define a scrollable?

**Comment by hirbod** [comment]
Hi @demchenkoalex, just FYI, I forked your project yesterday and just swapped everything related from FlatList to FlashList, and the results are incredibly great, specially on Android. I did it not do it a usable matter for this library, just in a quick for my project, so I can't provide a PR but I just wanted to let you know that it is really fast and works great! In order for this to be usable in the future, I would recommend to export a prop to provide a scrollable yourself or for a prop to switch the implementation. 

https://user-images.githubusercontent.com/504909/177118859-5e7f5dd8-d3b5-481d-9e48-2166f9748307.MOV
